src/rest.rs

```diff
@@ -1202,6 +1202,38 @@ fn handle_request(
                 .map_err(|err| HttpError::from(err.description().to_string()))?;
             http_message(StatusCode::OK, txid.to_hex(), 0)
         }
+        (&Method::POST, Some(&"txs"), Some(&"test"), None, None, None) => {
+            let txhexes: Vec<String> = String::from_utf8(body.to_vec())?
+                .split(',')
```

Comment:
From an API user perspective, does raw text hex separated by commas make more sense than a JSON array of strings?

Obviously this adds some unnecessary `[]` and `""` surrounding everything, but it seems a bit weird to just have raw stuff with random commas.

I guess this was just an extension of the existing push tx endpoint, and then commas were tacked on.

Maybe `\n` might be a better separator? Or `:` or something that doesn't make people wonder why it wasn't just a JSON to begin with.

idk, this isn't really a big issue, I think it's fine as is... but while the push tx endpoint feels fine, it just feels kind of weird when you add in multiple txes with commas.

Reply:
hmm, I was thinking that comma-separated strings were more framework-agnostic. But the response is JSON, so I guess we can assume users have that capability 😅